ClassHierarchyAndCollections/ClassHierarchyAndCollectionsTests/ClientTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ClassHierarchyAndCollections;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace ClassHierarchyAndCollectionsTests
+{
+    [TestClass]
+    public class ClientTests
+    {
+        [TestMethod]
+        public void Can_Create_Client()
+        {
+            var client = new Client("Bob", "911", 7, "Apache Helicopter", DateTime.Now, 5);
```

Comment:
"Apache Helicopter" is an interesting gender...This would be a good place for either an enum or good validation of allowed values.